images/universal/training/py312-cuda128-torch280/Dockerfile

```
@@ -54,20 +54,25 @@ RUN dnf config-manager \
     gcc-c++ \
     make \
     python3-devel \
     cmake \
     git \
+ && echo "/usr/local/nvidia/lib" >> /etc/ld.so.conf.d/nvidia.conf \
+ && echo "/usr/local/nvidia/lib64" >> /etc/ld.so.conf.d/nvidia.conf \
  && dnf clean all \
  && rm -rf /var/cache/dnf/*
 
 # Ensure CUDA_HOME points to the toolkit and nvcc is discoverable, then sanity check nvcc
 ENV CUDA_HOME=/usr/local/cuda \
```

Hadolint (2.14.0)

[error] Do not refer to an environment variable within the same `ENV` statement where it is defined.

(DL3044)